src/app/components/CurrencyAsset/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ * CurrencyAssets
+ *
+ */
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import { Asset } from 'types/asset';
+
+import { AssetsDictionary } from '../../../utils/dictionaries/assets-dictionary';
+
+interface CurrencyProps {
+  assetImg?: any;
+  assetSymbol: Asset;
+}
+const symbolMap = {
+  [Asset.RBTC]: 'rBTC',
+  [Asset.USDT]: 'rUSDT',
```

Comment:
Symbol map must have react node or html markup

```suggestion
  [Asset.RBTC]: <><small>r</small>BTC</>,
  [Asset.USDT]: <><small>r</small>USDT</>,
```